specification/containerinstance/resource-manager/readme.md

```diff
@@ -247,6 +247,13 @@ suppressions:
       - containerInstance.json
     where:
       - $.definitions.SecretReferenceVolume
+  - code: GetCollectionResponseSchema
+    reason: We do not return instanceView property in our ContainerGroups_List operation, however we do return this 
+            property for individual Get operations on container groups. 
```

Comment:
This is not the right pattern to implement. The list and point get responses should match for the same resource type. Pl fix it.